tests/bootstrap.php

```diff
@@ -18,6 +18,6 @@
 }
 
 /**
-* @var $loader ClassLoader
-*/
+ * @var ClassLoader
```

Comment:
I applied the change, lets hope it works :) I can squash the PR when merging :+1: